src/Upscale.hpp

```diff
@@ -123,6 +123,12 @@ class Upscale : public mfem::Operator
     virtual MixedMatrix& GetCoarseMatrix();
     virtual const MixedMatrix& GetCoarseMatrix() const;
 
+    virtual const mfem::SparseMatrix& GetVertexProjection() const { return coarsener_->get_Pu(); }
+
+    /// @todo I feel like needing this reflects some leaky abstractions
+    ///       or bad design in one way or another.
+    virtual const GraphCoarsen& GetGraphCoarsen() const { return coarsener_->get_GraphCoarsen_ref(); }
```

Comment:
Exposing this can be avoided using @chakshinglee's note on computing coarse constant using `P_vertex`.  Either in this class or outside utility function.
```
mfem::Vector GetCoarseConstant(const Upscale& upscale) const
{
         mfem::Vector ones = upscale.GetFineVector();
         return upscale.Restrict(ones);
}